transforms/csdl/preprocess_csdl_test_output.xml

```diff
@@ -726,6 +726,8 @@
           </Record>
         </Annotation>
       </Annotations>
+      <Annotations Target="microsoft.graph.authentication/requirements" />
```

Comment:
I think the readable and updatable annotations should show up here too. Yes? 
Any chance you can take a look at the changes on this PR? As annotations can show up in multiple places, using the approach there may be best to avoid a regression. 
https://github.com/microsoftgraph/msgraph-metadata/pull/497/files

Reply:
Yes, indeed they should. Thanks for that PR.

Reply:
@timayabi2020 Any chance you can rebase the changed to resolve the conflicts?